bundles/org.openhab.binding.revogismartstripcontrol/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,90 @@
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- SmartStrip Thing Type -->
+	<thing-type id="smartstrip">
+		<label>RevogiSmartStrip Binding Thing</label>
+		<description>A Binding to control Revogi Smartstrip</description>
+		<category>PowerOutlet</category>
+
+		<channel-groups>
+			<channel-group id="overallPlug" typeId="overallPlugActor"/>
+			<channel-group id="plug1" typeId="plugActor">
+				<label>Plug 1</label>
+			</channel-group>
+			<channel-group id="plug2" typeId="plugActor">
+				<label>Plug 2</label>
+			</channel-group>
+			<channel-group id="plug3" typeId="plugActor">
+				<label>Plug 3</label>
+			</channel-group>
+			<channel-group id="plug4" typeId="plugActor">
+				<label>Plug 4</label>
+			</channel-group>
+			<channel-group id="plug5" typeId="plugActor">
+				<label>Plug 5</label>
+			</channel-group>
+			<channel-group id="plug6" typeId="plugActor">
+				<label>Plug 6</label>
+			</channel-group>
+		</channel-groups>
+
+		<representation-property>serialNumber</representation-property>
+		<config-description>
+			<parameter name="serialNumber" type="text" required="true">
+				<label>Serial Number</label>
+				<description>Serial number of your smart strip.</description>
+			</parameter>
+			<parameter name="pollInterval" type="integer">
+				<label>Poll Intervall</label>
+				<default>60</default>
+				<description>How often (seconds) should the smart strip status be polled?</description>
+			</parameter>
+			<parameter name="ipAddress" type="text">
+				<label>IP Address</label>
+				<description>IP Address of your smart strip</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-group-type id="plugActor">
+		<label>Single Plug Actor</label>
+		<description>Switch a single plug and retrieve stats for it</description>
+		<channels>
+			<channel id="switch" typeId="single-plug"/>
+			<channel id="watt" typeId="watts"/>
+			<channel id="amp" typeId="amps"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="overallPlugActor">
+		<label>Overall Plug Actor</label>
+		<description>Switch all plugs</description>
+		<channels>
+			<channel id="switch" typeId="single-plug"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="single-plug">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
+		<description>Switch a single plug</description>
+	</channel-type>
+	<channel-type id="watts" advanced="true">
+		<item-type>Number</item-type>
+		<label>Watt</label>
+		<description>Contains the current watt value for the given plug</description>
+		<state readOnly="true" pattern="%.1f W" />
+	</channel-type>
+	<channel-type id="amps" advanced="true">
+		<item-type>Number</item-type>
+		<label>Amp</label>
+		<description>Contains the current Amp value for the given plug</description>
+		<state readOnly="true" pattern="%.1f A" />
+	</channel-type>
```

Comment:
Please apply openHab's [Units of Measurement](https://www.openhab.org/docs/concepts/units-of-measurement.html) framework wherever applicable. This allows user interfaces greater flexibility when it comes to rendering data from your binding.

Reply:
I updated the thing description, hope that's correct now. If not, I didn't understand what to do ;)